### PR TITLE
Change the function else condition so that can use in existing project

### PR DESCRIPTION
**Pull request by creativehassan**
I have changed the function because if someone want to use it in existing project and move all existing media to azure than can easily render media from azure all old and latest media files.

### CODE REVIEW

**Comment by allan23** [comment]
@creativehassan why change the function name?

**Comment by creativehassan** [comment]
@allan23  No function name change just inner body of function is modified for existing projects to use this plugin. 

Thanks

**Comment by allan23** [comment]
@creativehassan your code in your PR changes the name of the function from `windows_azure_storage_wp_get_attachment_url` to `edwards_azure_storage_wp_get_attachment_url()`.

**Comment by creativehassan** [comment]
Ah my bad let me revert it. and make it same

File: windows-azure-storage.php [hunk 1]
```diff
@@ -331,16 +331,19 @@ function windows_azure_storage_new_media_object( $args ) {
  *
  * @return string Returns metadata url.
  */
-function windows_azure_storage_wp_get_attachment_url( $url, $post_id ) {
+function edwards_azure_storage_wp_get_attachment_url( $url, $post_id ) {
```
Comment (allan23) [inline windows-azure-storage.php]: Function name needs updated.

**Review by allan23 — CHANGES_REQUESTED** [review]

**Comment by jeffpaul** [comment]
@creativehassan any updates on being able to revert the function name?

**Comment by jeffpaul** [comment]
@creativehassan any updates on being able to revert the function name?

**Comment by creativehassan** [comment]
Hi Jeff Paul, I was much busy some where else. I just change the name of the function.

**Comment by jeffpaul** [comment]
Moving this from the `4.3.0` release to `Future Release` to allow for more time to review the requested change here.

**Comment by jeffpaul** [comment]
@rickalee this is a pretty long-standing PR, any chance you or someone on the team can get this through review and off to merge?

**Comment by jeffpaul** [comment]
@rickalee @colegeissinger are you able to give this a final code review/test to get this into a release or should we close out this PR as no longer actionable/desired?

**Comment by jeffpaul** [comment]
Closing this as not currently actionable, but open to having this re-opened if someone is able to own the completion and review of this.

**Pull request closed without merging**